src/core/components/formGroup/FormGroup.module.css

```
@@ -0,0 +1,38 @@
+  padding-left: 30px;
+  padding-right: 30px;
+  gap: 16px;
+  border: none;
+}
+
+.titleContainer {
+  display: flex;
+  align-content: space-between;
+  flex-direction: column;
+
+  gap: 4px;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)